pom.xml

```diff
@@ -23,6 +23,12 @@
         <tycho-version>1.6.0</tycho-version>
         <tycho.scmUrl>scm:git:git://github.com:checkstyle/eclipse-cs.git</tycho.scmUrl>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.checkstyle.plugin.version>3.1.0</maven.checkstyle.plugin.version>
+        <maven.sevntu.checkstyle.plugin.version>1.37.0</maven.sevntu.checkstyle.plugin.version>
+        <!-- it should be a version of checkstyle that is compatible/compiled with sevntu -->
+        <maven.sevntu.checkstyle.plugin.checkstyle.version>
+            8.29
```

Comment:
https://checkstyle.org/releasenotes.html#Release_8.30

  Problematic Check is introduced in next version.
Please use there 8.35, latest released.

Reply:
please see how I did it at https://github.com/checkstyle/sonar-checkstyle/pull/316

Reply:
done